src/_data/styles/retro-cyberpunk.json

```
@@ -0,0 +1,6 @@
+{
+  "title": "Retro Cyberpunk",
+  "author": "Inès K.",
+  "stylesheet": "https://codepen.io/inescodes/pen/ExKEpxG.css",
+  "twitter": "inescodes",
```

```suggestion
  "twitter": "inescodes"
```

Just a little fixup so this is valid json 😊 don't worry, it's a common mistake!